domains/habiboff.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "habiboffdev",
+        "email": "mirzoupwork@gmail.com"
+    },
+    "record": {
+        "URL": "https://google.com"
```

Comment:
Denied.

Reply:
Reserving a domain for nothing is not allowed.